wordpress-mobile/WordPress-iOS
# Aztec announcement: Add confetti to 'New Editor Enabled' screen 🎉

This PR adds entirely superfluous confetti to the 'thank you' screen displayed when a user enables the beta of Aztec.

![confetti-1](https://user-images.githubusercontent.com/4780/27797651-559d973a-6006-11e7-9c57-81b5f4ae2757.gif)

To test:

* Do a build and fresh install of the app.
* When the Aztec beta alert appears, tap "Try It"
* Ensure everything looks fabulous.

Needs review: @jleandroperez
